From: "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>
Date: Tue, 7 Oct 2025 17:09:24 -0300
Subject: [PATCH 327/327] Linux 5.10.245-rt139 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=c3b3915fea9de2610b3c0148bd272660ef4273e8

Signed-off-by: Luis Claudio R. Goncalves <lgoncalv@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9ad53f..e0b93414dc30 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt139
